docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -17,7 +17,7 @@ Requirements
 
 To connect to Databricks Delta Lake, you need:
 
-* Tables written by Databricks Runtime 7.3 LTS, 9.1 LTS, 10.4 LTS, 11.3 LTS, and
```

Comment:
While at it, would it make sense to capture some tables created by 7.3 as static test resources and run tests against them?

Then we wouldn't change this file. Of course it's less about changing this file, but more about staying compatible with data out there. We should be able to read any Delta table, including those written by DBR << 7.

Reply:
BTW, I think most of the initial tables from the test resources used for testing `trino-delta-lake` are actually created with Databricks 7.3.

Trino can likely read as well tables written with non-LTS  version of Databricks, but we're not mentioning this here.
I think there's no loss / misleading information if we remove `7.3 LTS` from the documentation.

Reply:
I care less about docs, and more about testing coverage.
Tables created by DBR 7.3 do exist in the world and we must keep supporting them.
Is this test covered?